opennms-base-assembly/src/main/filtered/etc/collectd-configuration.xml

```diff
@@ -93,6 +131,8 @@
     </package>
 
 
+    <collector service="Cassandra21x" class-name="org.opennms.netmgt.collectd.Jsr160Collector"/>
+    <collector service="Newts" class-name="org.opennms.netmgt.collectd.Jsr160Collector"/>
```

Comment:
It's just a matter of taste. What do you think naming the services like `JMX-DC-Cassandra21x` and `JMX-DC-Newts`, so a user get an idea this service is used for JMX data collection?